src/components/index.ts

```diff
@@ -4,5 +4,5 @@ export { default as KeyboardAwareScrollView } from "./KeyboardAwareScrollView";
 export {
   default as KeyboardToolbar,
   DefaultKeyboardToolbarTheme,
-  KeyboardToolbarProps,
+  type KeyboardToolbarProps,
```

Comment:
Can it be `export type { KeyboardToolbarProps } from "./KeyboardToolbar"` as I did in https://github.com/kirillzyusko/react-native-keyboard-controller/pull/299?

Reply:
yes

Reply:
changed